src/machines.scss

```diff
@@ -102,10 +102,6 @@
   }
 }
 
-// radio buttons rendered vertically in pf-m-horizontal https://github.com/patternfly/patternfly/issues/5787
-.pf-v5-c-form__group-control.pf-m-inline {
-  .pf-v5-c-radio {
-    width: auto;
-    display: grid;
-  }
+#storage-pool-delete-volumes + label + span.pf-v5-c-check__body {
```

Comment:
This rule is a bit questionable, it would be good to have had a comment here

Reply:
I came up with a better rule and added a comment